src/Java.Interop.BootstrapTasks/Java.Interop.BootstrapTasks/JdkInfo.cs

```diff
@@ -150,6 +155,56 @@ void WriteMakeFragmentFile (string jarPath, string javacPath, string jdkJvmPath,
 			}
 		}
 
+		bool ValidateJdkPath (Version maxVersion, string java_home)
+		{
+			return ValidateJdkPath (maxVersion, java_home,
+					out var jar, out var javac, out var jdk, out var include);
```

Comment:
Looks like a perfect place to use [C#7 discards](https://docs.microsoft.com/en-us/dotnet/csharp/discards) :)